internal/handlers/pg/msg_listdatabases.go

```diff
@@ -73,10 +76,21 @@ func (h *Handler) MsgListDatabases(ctx context.Context, msg *wire.OpMsg) (*wire.
 				// Even though we run the query in a transaction, the current isolation level doesn't guarantee
 				// that the table is not deleted (see https://www.postgresql.org/docs/14/transaction-iso.html).
 				err = tx.QueryRow(ctx, "SELECT COALESCE(pg_total_relation_size($1), 0)", fullName).Scan(&tableSize)
-				if err != nil {
-					return lazyerrors.Error(err)
+				if err == nil {
+					sizeOnDisk += tableSize
+					continue
+				}
+
+				var pgErr *pgconn.PgError
+				if errors.As(err, &pgErr) {
+					switch pgErr.Code {
+					case pgerrcode.UndefinedTable:
+						// Table was deleted after we got the list of tables, just ignore it
```

Comment:
We can ignore this because all query or operation can handle the table not existing correct?